Makefile.kokkos

```diff
@@ -279,6 +284,18 @@ else
   KOKKOS_INTERNAL_OPENMPTARGET_FLAG := -fopenmp -foffload=nvptx-none
 endif
 
+# Set OpenACC flags.
+ifeq ($(KOKKOS_INTERNAL_COMPILER_PGI), 1)
+  KOKKOS_INTERNAL_OPENACC_FLAG := -acc
+else
+    ifeq ($(KOKKOS_INTERNAL_COMPILER_NVHPC), 1)
+      KOKKOS_INTERNAL_OPENACC_FLAG := -acc
+    else
+      #Assume GCC
+      KOKKOS_INTERNAL_OPENACC_FLAG := -fopenacc -foffload=nvptx-none
```

Comment:
hmm ... what about checking and printing if not gcc?

Reply:
Yes, sure! It is a good point, but what would be a good way to do that?
Maybe:
else
    $(error Makefile.kokkos: OpenACC is enabled but the compiler is neither NVHPC nor GCC (got version string $(KOKKOS_CXX_VERSION)) )
endif

Reply:
sounds good.

Reply:
The current OpenACC backend does not support GCC; please remove the GCC option lines and replace with the error message.

Reply:
Yes, I know. I wanted to keep GCC, although the current support of GCC for OpenACC C++ is not very good right now, maybe in a future can improve. Anyway, I can remove that for the moment, no problem.